controllers/imagejob/imagejob_controller.go

```diff
@@ -38,14 +38,14 @@ func Add(mgr manager.Manager) error {
 
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) reconcile.Reconciler {
-	return &Reconciler{
```

Comment:
revert Reconciler name changes

Reply:
If I revert the name to ImageJobReconciler I get the following lint error:

`exported: type name will be used as imagejob.ImageJobReconciler by other packages, and that stutters; consider calling this Reconciler (revive)`